<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextTool</class>
 <widget class="QWidget" name="TextTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>24</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Name</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Size</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Font</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Bold</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Italic</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Underline</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Text</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Propertie</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Size</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Font</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Bold</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>Italic</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>Underline</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>Text</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="MYUnitsSpinBox" name="heightSpinBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>19</y>
         <width>53</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>45</y>
         <width>112</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="underlineCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>117</y>
         <width>68</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Underline</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="italicCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>94</y>
         <width>46</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="boldCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>71</y>
         <width>43</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYUnitsSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>utils/myunitsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
